lldb/docs/index.rst

```diff
@@ -102,10 +102,13 @@ Additionally it builds:
 See the :doc:`LLDB Build Page <resources/build>` for build instructions.
 
 Discussions about LLDB should go to the `LLDB forum
-<https://discourse.llvm.org/c/subprojects/lldb>`__. Commit
+<https://discourse.llvm.org/c/subprojects/lldb>`__ or the ``lldb`` channel on
+the LLVM Discord server.
```

Comment:
Lets add the link to LLVM Discord server (https://discord.gg/HexWwB5z)

Reply:
Done.